docs/notebooks/units.ipynb

```diff
@@ -35,7 +35,7 @@
    "cell_type": "markdown",
```

Comment:
<p>IMO: °K -&gt; K</p><p>I've rarely seen the former used, maybe nitpicky</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>